### ISSUE TITLE
searching for values 

### ISSUE THREAD

**Issue by jamilo99** [microsoft/vscode-data-wrangler#351]
Type: <b>Bug</b>

Hi, 

I didn´t manage to find a "search value in the grid/dataframe" functionality. The usual ctrl+f doesn´t work and I don´t know what else I could do. Am I doing something wrong or this feature is still missing? Thanks!

# Behaviour

## Expected vs. Actual

XXX

## Steps to reproduce:

1. XXX

# Diagnostic data

-   Jupyter extension version: 2024.10.0
-   Python extension version: 2024.20.0
-   .NET Install Tool for Extension Authors extension version: Not installed
-   Python package dependencies:

```json
{
  "installed": {
    "pandas": "2.2.3"
  },
  "required": {
    "pandas": "1.2.0"
  },
  "unsatisfied": []
}
```

-   Entrypoint: JupyterCellStatusBar
-   Active mode: dataViewer

Extension version: 1.14.0
VS Code version: Code 1.95.3 (f1a4fb101478ce6ec82fe9627c43efbf9e98c813, 2024-11-13T14:50:04.152Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|13th Gen Intel(R) Core(TM) i5-1334U (12 x 2496)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.72GB (2.51GB free)|
|Process Argv|C:\\Users\\amalt\\OneDrive\\Desktop\\somatic_symptoms_app --crash-reporter-id 8816e96b-0bf3-4b17-b18d-c7772a39d317|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
dvdeprecation:31068756
dwnewjupyter:31046869
2f103344:31071589
nativerepl2:31139839
pythonrstrctxt:31112756
nativeloc1:31192215
cf971741:31144450
iacca1:31171482
notype1:31157159
5fd0e150:31155592
dwcopilot:31170013
stablechunks:31184530

```

</details>

**Comment by pwang347** [comment]
Hi @jamilo99, thanks for opening this issue! Does the filter functionality suit your needs? You can access this with the following button on the header cell overflow button:
<img width="188" alt="Image" src="https://github.com/user-attachments/assets/4ac465c2-97f2-40d1-9062-45c9dd525722" />

If you are interested to perform simple text searches for values in the grid, we will have a pre-release very soon (either today or next Monday) optimizing for this experience. I will update this thread again when it is released.

Thanks!

**Comment by pwang347** [comment]
Just wanted to update, the filter changes are available in pre-release version 1.15.1 if you'd like to try it out!

**Comment by pwang347** [comment]
Closing this issue due to inactivity and I believe it has been addressed by the latest pre-release update. Please feel free to reopen this if you have any additional followups!

**Comment by chklee30** [comment]
> Just wanted to update, the filter changes are available in pre-release version 1.15.1 if you'd like to try it out!

Where is this feature exactly? I came across this thread because I'm looking for a general "ctrl+F" functionality for data wrangler. Currently it seems like you can only search for values on chosen columns.

**Comment by chklee30** [comment]
Yes @pwang347 , searching for string (or float) values across the entire dataframe would be the use case.

**Comment by raffaelemancuso** [comment]
I would bet that there was this feature ... was it deleted or do I remember incorrectly?

Also see https://stackoverflow.com/questions/79031255/how-to-search-for-a-element-in-list-in-vs-code-data-wrangler-interactive-jupyte
